src/main/java/uk/gov/pay/adminusers/utils/email/EmailValidator.java

```diff
@@ -44,7 +44,8 @@
             "socialworkengland\\.org\\.uk",
             "ucds\\.email",
             "wmca\\.org\\.uk",
-            "york\\.ac\\.uk"
+            "york\\.ac\\.uk",
+            "nmni\\.com"
```

Comment:
the domains are in alphabetical order so far, so would be good to keep it that way